README.md

```diff
@@ -15,6 +15,29 @@ Dates are subject to change.
 
 The Release schedule is available also as a [JSON][] file.
 
+### Release Phases
+
+There are three phases that a Node.js release can be in: 'Current', 'Active
+Long Term Support (LTS)', and 'Maintenance'. Odd-numbered release lines are not
+promoted to LTS - they will not go through the 'Active LTS' or 'Maintenance'
```

Comment:
I thought that odd number releases went through a short maintenance period? is that no longer true?

Reply:
It is true according to the `schedule.json`- I think we've been purposely vague about it in the past. We can change that - maybe something like "release lines may receive critical fixes shortly after going EOL at the discretion of the Release team"? But I am not sure which is the best approach personally.